test/tools/libtesteth/FillJsonFunctions.cpp

```diff
@@ -52,19 +52,19 @@ json_spirit::mObject fillJsonWithTransaction(Transaction const& _txn)
 	return txObject;
 }
 
-json_spirit::mObject fillJsonWithStateChange(State const& _stateOrig, eth::State const& _statePost, std::vector<dev::eth::detail::Change> const& _changeLog)
+json_spirit::mObject fillJsonWithStateChange(State const& _stateOrig, eth::State const& _statePost, std::vector<dev::eth::Change> const& _changeLog)
 {
 	json_spirit::mObject oState;
 	if (!_changeLog.size())
 		return oState;
 
-	using changeKind = eth::detail::Change::Kind;
-	using changeStruct = eth::detail::Change;
-	eth::detail::ChangeLog changeLog = _changeLog;
+	using ChangeKind = eth::Change::Kind;
+	using ChangeStruct = eth::Change;
```

Comment:
This make no sense. You can use `Change` (because `using namespace dev::eth`), why you want to replace it with `ChangeStruct`.

Similar `ChangeKind` instead of `Change::Kind`?